Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -246,18 +264,6 @@
     if (!parseManifest(manifestData))
         return nil;
 
-    NSString *defaultLocale = [m_manifest objectForKey:defaultLocaleManifestKey];
-    m_defaultLocale = [NSLocale localeWithLocaleIdentifier:defaultLocale];
-
-    m_localization = [[_WKWebExtensionLocalization alloc] initWithWebExtension:*this];
-
-    m_manifest = [m_localization.get() localizedDictionaryForDictionary:m_manifest.get()];
-    ASSERT(m_manifest);
-
-    // FIXME: Handle Safari version compatibility check.
-    // Likely do this version checking when the extension is added to the WKWebExtensionController,
-    // since that will need delegated to the app.
```

Comment:
Was this FIXME handled by some other code change?

Reply:
No, but it is not needed any more. Safari still handles this internally.